app/api/tax.py

```diff
@@ -43,19 +45,20 @@ class TaxList(ResourceList):
 
     def query(self, view_kwargs):
         query_ = self.session.query(Tax)
-        if view_kwargs.get('id'):
-            query_ = query_.join(Event).filter(Event.id == view_kwargs['id'])
+        if view_kwargs.get('event_id'):
+            query_ = query_.join(Event).filter(Event.id == view_kwargs['event_id'])
         elif view_kwargs.get('identifier'):
             query_ = query_.join(Event).filter(Event.identifier == view_kwargs['identifier'])
         return query_
 
     def before_create_object(self, data, view_kwargs):
-        if view_kwargs.get('id'):
-            event = self.session.query(Event).filter_by(id=view_kwargs['id']).one()
+        if view_kwargs.get('event_id'):
+            event = self.session.query(Event).filter_by(id=view_kwargs['event_id']).one()
```

Comment:
@enigmaeth Most of the endpoints will be fixed in the process of solving https://github.com/fossasia/open-event-orga-server/issues/3802